cellmlmanip/model.py

```diff
@@ -502,7 +516,7 @@ def graph_with_sympy_numbers(self):
                 # Check if simplification removed dependencies on other variables, and if so remove the corresponding
                 # edges.
                 refs = self.find_variables_and_derivatives([rhs])
-                edges = list(graph.in_edges(equation.lhs))
+                edges = tuple(graph.in_edges(equation.lhs))
```

Comment:
Is this just for speed?

Reply:
yup iterating over immutable (i.e. tuples) or sets is quite a faster than over lists